Gruntfile.js

```diff
@@ -33,7 +33,7 @@ module.exports = function (grunt) {
             angular: {
                 src: ['bower_components/angular/angular.min.js',
                       'bower_components/angular-resource/angular-resource.min.js',
-                      'bower_components/angular-route/angular-route.min.js',
+                      'node_modules/angular-ui-router/release/angular-ui-router.min.js',
```

Comment:
@alihaghighatkhah Okay understood 👍 

@slasheks If this is possible, then sure, but not in this pull request.